src/main/webapp/WEB-INF/tags/instructor/comments/visibilityOptions.tag

```diff
@@ -20,7 +29,7 @@
     <div id="visibility-options${commentIdx}" class="panel panel-default"
         style="display: none;">
         <div class="panel-heading">Visibility Options</div>
-        <table class="table text-center text-color-black">
+        <table class="table text-center" style="color: #000;">
```

Comment:
Yupp, let's open another issue for this, text-color-black is definitely a better choice than RGB 000.